memory-store/migrations/000028_new_hybrid_search_implementation.up.sql

```diff
@@ -0,0 +1,142 @@
+CREATE OR REPLACE FUNCTION search_hybrid (
```

Comment:
does this work as is? you might need to drop the function first before this line, since you can't change the name of parameters of a function using `create or replace function`

so should add `DROP FUNCTION search_hybrid(uuid,text,vector,text[],uuid[],integer,double precision,double precision,jsonb,text);`